build/ical.js

```diff
@@ -6515,7 +6515,14 @@ ICAL.Event = (function() {
     },
 
     get endDate() {
-      return this._firstProp('dtend');
+      var end_date = this._firstProp('dtend');
+      if(end_date === null) {
+        end_date = this.startDate.clone();
+        if(this.startDate.isDate) {
+          end_date.addDuration(ICAL.Duration.fromSeconds(86400));
```

Comment:
Some minor style nits: Could you add spaces after `if` and use a camelCase variable name?
Also, maybe it would be faster to just add 1 day to the end date using the day property, instead of constructing a duration object just to add the duration?